modules/tripleliftBidAdapter.js

```diff
@@ -107,15 +111,18 @@ function _getSyncType(syncOptions) {
 function _buildPostBody(bidRequests) {
   let data = {};
   let { schain } = bidRequests[0];
-  data.imp = bidRequests.map(function(bid, index) {
-    return {
+  data.imp = bidRequests.map(function(bidRequest, index) {
+    let imp = {
       id: index,
-      tagid: bid.params.inventoryCode,
-      floor: _getFloor(bid),
-      banner: {
-        format: _sizes(bid.sizes)
-      }
+      tagid: bidRequest.params.inventoryCode,
+      floor: _getFloor(bidRequest)
+    };
+    if (bidRequest.mediaTypes.video) {
```

Comment:
this appears to be bad logic; multiformat outstream / banner / native units will be considered video units

Reply:
the effect is you don't bid on them

Reply:
Hi Pat,

Thanks for surfacing! We will review internally and update.

Reply:
Hi @patmmccann ,

I've opened [PR 5872](https://github.com/prebid/Prebid.js/pull/5872) to resolve this -- thanks again!